### ISSUE TITLE
Event name

### ISSUE THREAD

**Issue by mirkods** [hden/socketio-wildcard#25]
Hi there,
is there any way to know what the event name is?

Mirko

**Comment by mirkods** [comment]
I understood it from the tests but I think that should be documented for the future.

Congrats for the module
Mirko

**Comment by kaansoral** [comment]
What is it? (For the future stumblers too :)

**Comment by mirkods** [comment]
Sorry for the short explanation but yesterday I hadn't more time to describe it.
In my case i need to know which is the event name that is behind the wildcard in order to forward it based on the name.

I haven't found nothing on the docs but reading your tests I got it that I can retrive the name in the follow way:

```javascript
socket.on('*', function (packet) {
    const eventName = packet.data[0];
    const eventData = packet.data[1];
})
```

So I think this could be useful if it is a part of the docs.
Hope this time I explained better the case.
Mirko

**Comment by kaansoral** [comment]
Ah, it wasn't my test by the way, like you, I was looking for the same thing, and like you, I found it in the docs then forgot to write about it here :)

**Comment by iamchristopher** [comment]
Was also looking for this functionality. Thank you, @hden, for updating the README but might you also publish the changes to npm?

**Comment by hden** [comment]
Published as `v0.4.0`
